_zh-cn/overviews/scala3-book/scala-features.md

```diff
@@ -4,15 +4,10 @@ type: chapter
 description: This page discusses the main features of the Scala 3 programming language.
 num: 2
 previous-page: introduction
-next-page: 
-
-scala3: true
-partof: scala3-book
-overview-name: "Scala 3 — Book"
-layout: multipage-overview
-permalink: "/zh-cn/scala3/book/:title.html"
```

Comment:
again, it seems lines 9-13 is necessary across each page in the book

Reply:
add.